apps/storage/api/attachments/presign.get.ts

```diff
@@ -0,0 +1,70 @@
+import { PutObjectCommand } from '@aws-sdk/client-s3';
+import { getSignedUrl } from '@aws-sdk/s3-request-presigner';
+import { db } from '@uninbox/database';
+import { and, eq } from '@uninbox/database/orm';
+import { orgMembers, orgs } from '@uninbox/database/schema';
+import { nanoId } from '@uninbox/utils';
+import { z } from 'zod';
+
+const bodySchema = z.object({
+  orgSlug: z.string(),
+  filename: z.string()
+});
+/**
+ * Returns a presigned URL for uploading an attachment to S3
+ */
+export default eventHandler({
+  onRequest: [loggedIn],
+  async handler(event) {
+    const inputValidation = await getValidatedQuery(event, (body) =>
+      bodySchema.safeParse(body)
+    );
+    if (!inputValidation.success) {
+      setResponseStatus(event, 400);
+      return send(event, 'Invalid input');
+    }
+    const data = inputValidation.data;
+
+    const userId = +event.context.user.id;
```

Comment:
Why do we need to cast the userId here with the `+`?

Reply:
db sometimes responds with a string instead of number, and when storing in redis it also gets transformed on occaasion
should be fixed, but I just got into the habit of casting it to make sure